### PR TITLE
Qencoder compatibility patch

### PR DESCRIPTION
**Pull request by natis1**
Add ffmpeg splitting, add --quiet to stop tqdm from loading, make scenedetect and tqdm not strictly required, add setup-minimal for av1an-minimal, a version with much fewer deps, when using --quiet av1an exposes the number of frames that have been encoded

### CODE REVIEW

File: av1an/manager/Counter.py [hunk 1]
```diff
@@ -1,5 +1,10 @@
 from multiprocessing.managers import BaseManager
-from tqdm import tqdm
+has_tqdm = 1
```
Comment (master-of-zen) [inline av1an/manager/Counter.py:2]: yeah, this can be prettier

File: setup-minimal.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import setuptools
+
+REQUIRES = [
+    'numpy',
+    'psutil',
+    'scipy',
+    'matplotlib',
```
Comment (master-of-zen) [inline setup-minimal.py:7]: do we really need matplotlib?

File: av1an/scenedetection/pyscene.py [hunk 3]
```diff
@@ -3,10 +3,14 @@
...
-from scenedetect.video_manager import VideoManager
-from scenedetect.frame_timecode import FrameTimecode
+scenedetect = 1
+try:
+    from scenedetect.detectors import ContentDetector
+    from scenedetect.scene_manager import SceneManager
+    from scenedetect.video_manager import VideoManager
+    from scenedetect.frame_timecode import FrameTimecode
+except ImportError:
+    scenedetect = 0
```
Comment (n9Mtq4) [inline av1an/scenedetection/pyscene.py:13]: use True/False here too

File: av1an/scenedetection/aom_kf.py [hunk 4]
```diff
@@ -8,7 +8,11 @@
 from subprocess import PIPE, STDOUT
 
 import cv2
-from tqdm import tqdm
+has_tqdm = 1
+try:
+    from tqdm import tqdm
+except ImportError:
+    has_tqdm = 0
```
Comment (n9Mtq4) [inline av1an/scenedetection/aom_kf.py:15]: True/False

**Comment by natis1** [comment]
Actually using true/false is suboptimal. the recommended way according to pycharm is to set the thing you would be importing to a None type, which I am doing now.

**Review by n9Mtq4 — APPROVED** [review]

**Comment by master-of-zen** [comment]
@natis1 :1st_place_medal:

**Pull request merged**